src/components/Cloud/Pull-Request-Item.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * Display status about a single Pull Request.
+ *
+ * @param {String} date       Date that the pull request was opened.
+ * @param {Number} id         GitHub Pull Request identifier.
+ * @param {String} link       Link to the pull request on GitHub.
+ * @param {String} status     Un-mapped status of the pull request
+ * @param {String} statusText Formatted string describing the current status of the pull request,
+ * @param {String} title      Title of the pull request.
+ */
+const PullRequestItem = ( { date, id, link, status, statusText, title } ) => {
+	const parsedDate = new Date( date );
+	return <li className="pull-request-item">
+		<div className="pull-request-item__info" >
+			<p className="pull-request-item__title">
```

Comment:
Guessing this is a `<p>` because of default WP styling? I'd like this to adhere to a11y guidelines, eg. if it's a title lets use the appropriate `<hN>`  tag.

Reply:
It is, the heading classes in the dashboard widgets page are arcanely written and require a crazy amount of specificity to override. Wasn't thinking of a11y as much as I should've so will correct these.